opencti-platform/opencti-graphql/src/domain/backgroundTask.js

```diff
@@ -107,7 +107,7 @@ const buildQueryFilters = async (context, user, rawFilters, search, taskPosition
   }
   const newFilters = buildQueryFiltersContent(adaptedFilterGroup);
   // Avoid empty type which will target internal objects and relationships as well
-  const types = newFilters.filters.filter((f) => f.key === TYPE_FILTER)?.values ?? [ABSTRACT_STIX_DOMAIN_OBJECT];
+  const types = newFilters?.filters.filter((f) => f.key === TYPE_FILTER)?.values ?? [ABSTRACT_STIX_DOMAIN_OBJECT];
```

Comment:
There is still a issue here : `newFilters?.filters.filter((f) => f.key === TYPE_FILTER)` is an array, we can't get the values with `.values` which only returns an iterator of the array.

Reply:
fixed